plugins/catalog/src/components/ComponentRemovalDialog/ComponentRemovalDialog.tsx

```diff
@@ -32,6 +32,7 @@ import React, { FC } from 'react';
 import { useAsync } from 'react-use';
 import { AsyncState } from 'react-use/lib/useAsync';
 import { catalogApiRef } from '../../api/types';
+import { alertApiRef } from '../../../../../packages/core-api/src/apis/definitions/AlertApi';
```

Comment:
oops, should be a package import - we should have a lint rule or something that forbids this